src/hotspot/cpu/riscv/frame_riscv.cpp

```diff
@@ -154,7 +154,7 @@ bool frame::safe_for_sender(JavaThread *thread) {
 
       sender_unextended_sp = sender_sp;
       sender_pc = (address) *(sender_sp - 1);
```

Comment:
We have `-1` here, and below `-2` is replaced as frame::metadata_words, seems it's still bit confusing to me. Similar comments to several other places.
But I'm not quite sure what should be the better way to fix it.

Reply:
Thanks for having a look. Yeah, I also witnessed that when making this change and that's also partially the reason when I used constant number 2 instead of `frame::metadata_words`. But I didn't touch this `-1` as similar issue is also there for aarch64 and x86 platforms and there doesn't seem to be a proper enum for this purpose. Maybe another separate PR if we want to fix that for all?